### ISSUE TITLE
Still exploring this?

### ISSUE THREAD

**Issue by ahasani** [nturley/scxml-hdl#1]
Hi Neil,

Interesting idea Are still on this?

Cheers

**Comment by nturley** [comment]
Nope. It still feels like a good idea though. Someday I'll circle back around on it. If you want to try it I'd be happy to contribute.

**Comment by ahasani** [comment]
It is indeed, for me it's still far down the road mate. I am working on JS implementation to learn the algorithm and exploring data model representation. Also learning from the great SCION by @jbeard4. Then I will work on a low level runtime and or transpiler. This supposedly can be turned into what Dr. Samek has done with his Quantum Leap. All these with the grossly underrated SCXML
